openwisp_controller/config/admin.py

```diff
@@ -496,7 +496,7 @@ def get_extra_context(self, pk=None):
                     'admin:get_relevant_templates', args=['org_id']
                 ),
                 'commands_api_endpoint': reverse(
-                    'openwisp_controller:api_device_command_list_create',
+                    'controller_connection:api_device_command_list_create',
```

Comment:
look at this issue: https://github.com/openwisp/openwisp-controller/issues/467
Do you think it's possible to make the namespace of these views similar to what is described there?

Reply:
Following the work done in https://github.com/openwisp/openwisp-controller/issues/467 and what is being done in https://github.com/openwisp/openwisp-controller/pull/474/, we should make sure the naming of these URLs follows the same style, eg: `connection_api:device_command_list`